modules/autocomplete-feed/module.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": [
+    "filter",
+    "map",
+    "sort",
+    "util"
+  ],
+  "keywords": [
```

Comment:
fyi, keywords aren't actually needed here (despite every module having them). The ones used by the docs are defined elsewhere